contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -482,6 +510,27 @@ NXmpes(NXobject):
         doc: |
           Voltage applied to sample and sample holder.
     (NXdata):
+      doc: |
+        An NXdata field containing a view on the measured data.
+        This NXdata field contains a collection of the main relevant fields (axes).
+        In NXmpes, it is required to provide an energy axis.
+        The other data fields ideally should be named according to conventions
+        to ensure compatibility. We recommened the following field names
+        for common data fields:
+
+        - **kx**: Calibrated x axis in k-space. Unit category: NX_ANY (e.g., 1/Angström).
+        - **ky**: Calibrated y axis in k-space. Unit category: NX_ANY (1/Angström).
+        - **kz**: Calibrated z axis in k-space. Unit category: NX_ANY (1/Angström).
+        - **delay**: Calibrated delay time. Unit category: NX_TIME (s).
+        - **polarization_angle**: Linear polarization angle of the incoming or
+          outgoing beam. This could be a link to
+          /entry/instrument/beam/incident_polarization_angle or
+          /entry/instrument/beam/final_polarization_angle if they exist.
+          Unit category: NX_ANGLE (° or rad)
+        - **ellipticity**: Ellipticity of the incoming or outgoing beam.
+          Could be a link to /entry/instrument/beam/incident_ellipticity or
+          /entry/instrument/beam/final_ellipticity if they exist.
+          Unit category: NX_ANGLE (° or rad)
```

Comment:
Intentionally did not add `sample_bias `and `temperature` since they are not particular for MPES. Kinetic_energy is already part of `energy` with `@type=kinetic`.

I am not so familiar with spin-resolved measurements, so any suggestions for the spin axis?